zebra/kernel_netlink.c

```diff
@@ -524,6 +524,31 @@ void netlink_parse_rtattr_nested(struct rtattr **tb, int max,
 	netlink_parse_rtattr(tb, max, RTA_DATA(rta), RTA_PAYLOAD(rta));
 }
 
+int rta_addattr_l(struct rtattr *rta, unsigned int maxlen, int type,
```

Comment:
it seems like the only thing that can be returned here is `0` ... is this correct?